### PR TITLE
Remove FRT E2E tests

### PR DESCRIPTION
**Pull request by eduardoumpierre**
#### Changes proposed in this Pull Request

Remove the FRT E2E tests as this test suite was impacting all other tests. The main issue was that the ruleset_config wasn't being updated, which led to the checkout being blocked even with a basic protection level set. p1688990882294179-slack-C01B8KNUYSW

#### Testing instructions

* E2E tests should be passing

-------------------

- [x] Run `npm run changelog` to add a changelog file, choose `patch` to leave it empty if the change is not significant. You can add multiple changelog files in one PR by running this command a few times. 
- [ ] Covered with tests (or have a good reason not to test in description ☝️)
- [ ] Tested on mobile (or does not apply)

**Post merge**

- [ ] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) following [these instructions](https://github.com/Automattic/woocommerce-payments/wiki/How-to-write-good-manual-testing-scenarios) : _Add link here / 'QA Testing Not Applicable'_
- [ ] Add or update [critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Critical-flows) and [testing instructions for critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Testing-instructions-for-critical-flows), if applicable.
- [ ] Add what's changed (description, screenshot, demo videos etc.) to the release announcement post, if applicable.

### CODE REVIEW

**Comment by botwoo** [comment]
### Test the build 

#### Option 1. Jetpack Beta

- Install and activate [Jetpack Beta](https://jetpack.com/download-jetpack-beta/).
- Use this build by searching for PR number `6733` or branch name `fix/e2e-tests-frt` in your-test.site/wp-admin/admin.php?page=jetpack-beta&plugin=woocommerce-payments          

#### Option 2. Jurassic Ninja - available for logged-in A12s

:rocket: [Launch a JN site with this branch](https://jurassic.ninja/create/?jetpack-beta&shortlived&nojetpack&woocommerce&branches.woocommerce-payments=fix/e2e-tests-frt) :rocket:

:information_source: Install this [Tampermonkey script](https://github.com/Automattic/woocommerce-payments/tree/develop/bin/wcpay-live-branches) to get more options.

---

Build info: 

- Latest commit: 70cc727e0089208cb7012423ac5a93dc488f1c74
- Build time: 2023-07-12 18:24:16 UTC

_Note: the build is updated when a new commit is pushed to this PR._

File: tests/e2e/utils/payments.js [hunk 1]
```diff
@@ -218,8 +218,7 @@ export async function confirmCardAuthentication(
  */
```
Comment (jessepearson) [inline tests/e2e/utils/payments.js:217]: The param above this line can be removed.

**Review by jessepearson — APPROVED** [review]
Per our discussion in Slack ( p1689179140354599/1688990882.294179-slack-C01B8KNUYSW ) going to go ahead and approve this pending the update to the docblock mentioned. I have double checked it against [the initial PR](https://github.com/Automattic/woocommerce-payments/pull/6357/files) and that's the only line that is differing between the two.

File: tests/e2e/utils/payments.js (hunk 1 above)
Comment (eduardoumpierre) [reply tests/e2e/utils/payments.js:217]: Param removed on [70cc727](https://github.com/Automattic/woocommerce-payments/pull/6733/commits/70cc727e0089208cb7012423ac5a93dc488f1c74). Thanks for catching it!

**Pull request merged**